internal/decoder/api/decoder_arm64.go

```diff
@@ -30,7 +30,7 @@ var (
 
 
 func init() {
-    // whe in aarch64. we enable all optimize
+    // when in aarch64, we enable all optimization
```

Comment:
```suggestion
    // when in arch64, we enable all optimization
```
Typo?

But shouldn't it be arm64?

Reply:
aarch64 is the alias of arm64.